src/UI/Component/Panel/Listing/Listing.php

```diff
@@ -22,4 +22,18 @@ public function getTitle();
 	 * @return \ILIAS\UI\Component\Item\Group[]
 	 */
 	public function getItemGroups();
+
+	/**
+	 * Sets the action drop down to be displayed on the right of the title
+	 * @param \ILIAS\UI\Component\Dropdown\Standard $actions
+	 * @return Listing
+	 */
+	public function withActions(\ILIAS\UI\Component\Dropdown\Standard $actions);
```

Comment:
Ok, I see that I thought this was about items. So my arguments apply to the items.